src/screens/ToDoList/styles.scss

```diff
@@ -0,0 +1,78 @@
+  $main-color : #40e0d0;
+
+.container {
+  border: 3px solid $main-color;
+  border-radius: 18px;
+  width: 400px;
+  min-height: 400px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 100px;
+  background-color:white;
+  display: flex;
+  flex-direction: column;
+  box-shadow: 20px 10px 10px 15px #000000, 30px 10px 5px -90px #000000;
+}
+
+.text-decoration {
+  color:black;
+  display: flex;
+  letter-spacing: 2px;
+}
+
+.line {
+  border-bottom: 2px solid $main-color;
+  margin: 0 15px;
+  box-shadow: 1px 1px 1px 1px #afaa10, 0px 0px 0px 0px #1f8e22;
+
+}
+
+.title {
+  align-items: center;
+  text-align: center;
+  color: lightslategray;
+  font-stretch: expanded;
+  background-color: $main-color;
+  letter-spacing: 2px;
+}
+
+.clear-button {
```

Comment:
You should create `button` mixin and put there all basic styles for button like `cursor`, `outline` etc.

Reply:
I will do this when code will be correct